modules/oneVideoBidAdapter.js

```diff
@@ -282,7 +289,16 @@ function getRequestData(bid, consentData, bidRequest) {
       bidData.regs.ext.us_privacy = bidRequest.uspConsent
     }
   }
-
+  if (bid.params.video.e2etest) {
+    bidData.imp[0].bidfloor = null;
+    bidData.imp[0].video.w = 300;
+    bidData.imp[0].video.h = 250;
```

Comment:
To clarify - should these be hard-coded?  Or should they be data-points about the current page's setup?

Reply:
Hi @jsnellbaker,
The e2etest param is supposed to enforce/override the client parameters to ensure that the outbound bid-request will receive the test creative response from our oneVideo ad-server.
Until now, when we wanted to do e2e testing with a partner we were dependent on them to implement several changes to the adUnit config. In most cases, the turnaround was slow as either typos would appear, incorrect values and the whole back and forth is time consuming.
By forcing these parameters the can test their Player integrations in a self served manner and reduce the time to go live.